### PR TITLE
v5.31.1 Changelog 

### PR DESCRIPTION
**Pull request by amyblais**
To let customers know about an upcoming dot release.

### CODE REVIEW

File: source/administration/changelog.md [hunk 1]
```diff
@@ -6,7 +6,11 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)
 
-**Release Day: 2021-01-16**
+- **v5.31.1, release day TBD**
+  - Fixed an issue where the npm packaging did not allow overwriting a version. There was a problem with the mattermost-redux version that was published for v5.31.1 and downstream projects were affected by this.
```
Comment (metanerd) [inline source/administration/changelog.md:10]:
```suggestion
  - Fixed an issue where mattermost-redux was not including the built code for v5.31.0 and downstream projects were affected by this.
```

**Review by metanerd — APPROVED** [review]

File: source/administration/changelog.md [hunk 2]
```diff
@@ -6,7 +6,11 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.31 - [ESR](https://docs.mattermost.com/administration/extended-support-release.html)
 
-**Release Day: 2021-01-16**
+- **v5.31.1, release day TBD**
+  - Fixed an issue where mattermost-redux was not including the built code for v5.31.0 and downstream projects were affected by this.
+  - Fixed an issue where the About Mattermost modal didn't display the correct Database Schema version.
```
Comment (justinegeffen) [inline source/administration/changelog.md:11]:
```suggestion
  - Fixed an issue where the **About Mattermost** modal didn't display the correct Database Schema version.
```

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Just one formatting suggestion, otherwise it's good from my side. :)

File: source/administration/changelog.md [hunk 3]
```diff
@@ -6,7 +6,11 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
...
+- **v5.31.0, released 2021-01-16**
+  - Original 5.30.0 release
```
Comment (elyscape) [inline source/administration/changelog.md:13]: This should read "Original 5.31.0 release".

Comment (amyblais) [reply source/administration/changelog.md]: Thank you!

**Review by larkox — APPROVED** [review]

**Pull request merged**